src/system/PageLockModule/Api/LockingApi.php

```diff
@@ -198,7 +198,8 @@ public function requireLock($lockName, $lockedByTitle, $lockedByIPNo, $sessionId
         $count = $this->repository->getActiveLockAmount($lockName, $theSessionId);
 
         $now = time();
-        $expireDate = new \DateTime($now + self::PAGELOCKLIFETIME);
+        $expireDate = new \DateTime();
+        $expireDate->setTimestamp($now + self::PAGELOCKLIFETIME);
```

Comment:
Oversaw that (confused `$now` with `'now'`) :confused:  . @MrMontesa reported it in #3091 already.